library/Zend/Mail/Storage/Part.php

```diff
@@ -122,6 +126,10 @@ public function __construct(array $params)
      */
     public function isMultipart()
     {
+        if (!isset($this->contentType)) {
```

Comment:
this part does not change behavior, but it avoids exception and try-catch on missing header. Change is ok.